lib/modules/manager/nix/artifacts.ts

```diff
@@ -22,9 +22,7 @@ export async function updateArtifacts({
     return null;
   }
 
-  let cmd = `nix \
-    --extra-experimental-features nix-command \
-    --extra-experimental-features flakes `;
+  let cmd = `nix --extra-experimental-features 'nix-command flakes' `;
```

Comment:
I'm not fully sure, but single quotes are failing on windows. you should use `quote` from `shlex` instead

Reply:
I don't think that matters much right now as nix' windows support is pretty bare bones and only compiles with some unmerged changes.